hw1/arraySort/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+class Program
```

Comment:
```suggestion
static class Program
```
Кроме того, можно использовать [top-level statements](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/top-level-statements) и класс тут вообще не писать